java/src/jmri/jmrit/logix/WarrantBundle_cs.properties

```diff
@@ -0,0 +1,448 @@
+TrackerBlockOccupied= Sledov\u00e1n\u00ed "{0}" m\u016f\u017ee b\u00fdt nep\u0159esn\u00e9, sousedn\u00ed blok "{1}" je ji\u017e obsazen.
+TrackerNoCurrentBlock= Um\u00edst\u011bn\u00ed "{0}" je nezn\u00e1m\u00e9. Posledn\u00ed zn\u00e1m\u00e9 um\u00edst\u011bn\u00ed na "{1}" je neobsazeno.
+TrackingStopped = Sledov\u00e1n\u00ed zastaveno.
+PossibleLocation= Vyberte mo\u017en\u00e9 um\u00edst\u011bn\u00ed.
+TrackerLocationLost= Um\u00edst\u011bn\u00ed "{0}" nen\u00ed zn\u00e1m\u00e9.
+TrackerLocation = Sledov\u00e1n\u00ed zastaveno. Um\u00edst\u011bn\u00ed "{0}" nen\u00ed zn\u00e1m\u00e9. 
+MultipleTrackers = Do bloku "{0}" m\u016f\u017ee vjet jeden z n\u00e1sleduj\u00edc\u00edch vlak\u016f.\nPros\u00edm vyberte kter\u00fd z vlak\u016f m\u00e1 vjet do "{0}".
+TrackerStopped  = Sledov\u00e1n\u00ed pro vlak "{0}" ukon\u010deno.
+blockInUse      = Blok "{1}" je v sou\u010dasnosti obsazen Stopa\u0159em "{0}".
+startTracker    = Stopa\u0159 za\u010dal pro "{0}" v bloku "{1}".
+duplicateName   = Stopa\u0159 s n\u00e1zvem "{0}" je ji\u017e pou\u017eit.
+noTrainName     = Zadejte n\u00e1zev tohoto vlaku.
+trainReport     = Vlak "{0}": {1}
+warrantStart    = Rozkaz "{1}" ud\u011blen vlaku "{0}" v bloku "{2}". {3} 
+startDark	= Po\u010d\u00e1te\u010dn\u00ed blok bez detekce obsazen\u00ed.
+startUnoccupied	= Po\u010d\u00e1te\u010dn\u00ed blok neobsazen.
+startAppendage	= {0} {1} v re\u017eimu {2}.
+warrantEnd      = Rozkaz "{1}" ukon\u010den pro vlak "{0}". Nedos\u00e1hl c\u00edlov\u00e9ho bloku "{2}".
+warrantComplete	= Rozkaz "{1}" dokon\u010den pro vlak "{0}". Nyn\u00ed obsazuje blok "{2}".
+warrantAnnull   = Rozkaz "{1}" zru\u0161ena pro vlak "{0}".
+warrantAbort    = Vlak "{0}" P\u0158ERU\u0160EN. (rozkaz {1})
+LearnAbortOccupied= Zaznamen\u00e1v\u00e1n\u00ed rozkazu "{1}" P\u0158ERU\u0160ENO. Blok {0} vp\u0159edu obsazen.
+controlChange   = Ovl\u00e1d\u00e1n\u00ed nastaveno na {2} pro vlak "{0}" - bylo {1}
+controlFailed   = Ovl\u00e1d\u00e1n\u00ed {2} selhalo pro vlak "{0}" - {1}
+LearnInvalidControl= Nemohu zastavit nebo obnovit skript b\u011bhem z\u00e1znamu {0}.
+LearnError      = Nemohu pokra\u010dovat v U\u010d\u00edc\u00edm se re\u017eimu: {0}
+NoPaths         = \u017d\u00e1dn\u00e1 cesta v bloku "{0}"
+PickAPath       = Vyberte cestu
+Origin          = Po\u010d\u00e1tek: {0}
+Ramp		= Br\u017ed\u011bn\u00ed do zastaven\u00ed
+Halt            = Zastavit
+Resume          = Obnovit
+Retry           = P\u0159esunout se do dal\u0161\u00edho bloku
+Abort           = P\u0159eru\u0161it
+Debug		= Ladit
+Idle            = Ne\u010dinn\u00fd
+Idle1           = Ne\u010dinn\u00fd - {0}
+Mark            = Vstupn\u00ed blok
+Skip            = P\u0159esko\u010den\u00fd blok
+# Note: some values contain formatting between pairs of {}'s so do NOT translate the word "number"
+SCWStatus       = Blok: {0} (rejst\u0159\u00edk {1}) Rychlost: {2, \u010d\u00edslo, #.#} N\u00e1v\u011bstidlo: {3} zobrazuje {4}
```

Comment:
`{2, \u010d\u00edslo, #.#}` should be `{2, number, #.#}` (see the comment on prior line). In this case, "number" is formatting instruction, not text displayed to a person.

This is what is causing the AppVeyor CI tests to fail with:
```
Command executed with exception: [resourceCheck] messageformat check: Unable to create MessageFormat instance unknown format type: \u010d\u00edslo (C:\projects\jmri\java\src\jmri\jmrit\logix\WarrantBundle_cs.properties:256:Blok: {0} (rejst\u0159\u00edk {1}) Rychlost: {2, \u010d\u00edslo, #.#} N\u00e1v\u011bstidlo: {3} zobrazuje {4})
```